docs/guides/architecture/configuration-files.md

```diff
@@ -4,19 +4,82 @@ sidebar_position: 1
 
 # Configuration Files
 
-Since Tauri is a toolkit for building applications there can be a lot of places to configure project settings. Some common files that you may run across are `tauri.conf.json`, `package.json`, `Cargo.toml`, and more. You can use this page as a reference to understand the differences between each of these and where to configure the relevant settings for your project.
+Since Tauri is a toolkit for building applications there can be a lot of places to configure project settings. Some common files that you may run across are `tauri.conf.json`, `package.json` and `Cargo.toml`. You can use this page as a reference to understand the differences between each of these and where to configure the relevant settings for your project.
 
 ## `tauri.conf.json`
 
 This is the file used by the Tauri process. You can define the build settings (such as the [command run prior to `tauri build`][before-build-command] or [`tauri dev`][before-dev-command]), set the [name and version of your app][package-config], [control the Tauri process][tauri-config], and any plugin settings. Find all of the options for this config file in the [`tauri.conf.json` API reference].
 
 ## `Cargo.toml`
 
-Cargo's manifest file. You can declare Rust crates your app depends on, metadata about your app, and much more. For the full reference see Cargo's Manifest Format.
+Cargo's manifest file. You can declare Rust crates your app depends on, metadata about your app, and much more. If you do not intend to do a lot of backend development using Rust for your app then you probably won't be changing it very much, but it's important to know it exists and what it does.
+
+An example of a barebones `Cargo.toml` file for a Tauri project might look a little something like this:
+
+```toml
+[package]
```

Comment:
Are these the values that get populated when you run `tauri init`?

Reply:
In Cargo.toml, yes. The package.json is much more framework dependent so it's harder to know which parts should be put in there as an example